src/client/views/index.html

```diff
@@ -35,6 +35,8 @@
 	<script src="https://appsforoffice.microsoft.com/lib/1.1/hosted/office.js"></script>
 	<app class="app">
 		<div id="progress" class="ms-progress-component">
+			<div class="ribbon left"></div>
```

Comment:
I believe this is necessary to populate the div with the environment contents on page load. In `index.script.tx` and `run.script.ts`, these divs are filled via `textContent`, and the user will see the left and right ribbons with the appropriate environment when loading the app. Not sure how I would accomplish the same with `before` and `after` (also have not used them before).